src/content/getusermedia/getdisplaymedia/js/main.js

```diff
@@ -7,9 +7,14 @@
  */
 'use strict';
 
-// Polyfill in Firefox.
-// See https://blog.mozilla.org/webrtc/getdisplaymedia-now-available-in-adapter-js/
-if (adapter.browserDetails.browser == 'firefox') {
+const sharingPreference = document.getElementById('preference');
```

Comment:
How about `preferredDisplaySurface` instead of `sharingPreference` so that it's clear?

Reply:
and a better name for the element